lib/react_client.dart

```diff
@@ -217,11 +218,6 @@ final ReactDartInteropStatics _dartInteropStatics = (() {
         return component;
       });
 
-  InteropContextValue handleGetChildContext(Component component) =>
```

Comment:
If this is public API, it should be restored since a `ReactDartInteropStatics2` class is being created in #161.

Once all of the new context stuff is implemented in there, `ReactDartInteropStatics2.handleGetChildContext` should be deprecated with a comment explaining what to use instead.